src/app/pages/biotools-repo-page/biotools-repo-page.component.ts

```diff
@@ -386,4 +388,57 @@ export class BiotoolsRepoPageComponent {
     const downloadinfo = note.split(';');
     return downloadinfo[3].replace('config:','');
   }
+
+  private updateIcons(biotoolsApplication: BiotoolsApplication[]): void {
+
+    for(var application of biotoolsApplication) {
+      application.icons = { cloud: false, book: false, binder:false, globe: false, window: false };
+
+      console.log("Checking if application is of type Notebook");
+      if (application.toolType.length > 0 && application.toolType.indexOf('Notebook') >= 0) {
+        application.icons.book = true;
+      } else {
```

Comment:
I think we should not have an `else` here. I suspect application cam be of type Notebook and additionally deployable on Binder or as a VM.

Reply:
ok, I have done the change and the one felix spotted